lib/browser/browser.ts

```diff
@@ -333,17 +350,26 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
       idToken = (await this.tokenManager.getTokens()).idToken as IDToken;
     }
   
+ 
+    if (revokeRefreshToken && typeof refreshToken === 'undefined') {
+      refreshToken = (await this.tokenManager.getTokens()).refreshToken as RefreshToken;
+    }
+
     if (revokeAccessToken && typeof accessToken === 'undefined') {
       accessToken = (await this.tokenManager.getTokens()).accessToken as AccessToken;
     }
   
     // Clear all local tokens
     this.tokenManager.clear();
```

Comment:
We may want to emit token change events when clear all tokens to update authState properly. Can be done in another PR.